### DISCUSSION TITLE
Pesquisa sobre o mercado LGBT

### DISCUSSION THREAD

**Discussion by wagnerbeethoven** [frontendbr/forum#1134] [Discussão]
Olá galerinha, tudo beleza? Sabemos que precisamos tomar algumas atitudes precisamos levantar dados, por isso gostaria de divulgar essa pesquisa.

Sei que o mercado é bem preconceituoso, não sei se há LGBTs aqui (também não sei se pode fazer esse tipo de divulgação).

**Pesquisa: Termômetro do Mercado de Trabalho pra LGBTs**

http://bit.ly/mercadoLGBT

Além de responder essa pesquisa, na empresa de vocês, como é o trato com gays, lésbicas, bisexuais, travestis e transgeneros?

:rainbow:

**Comment by fernandoRj** [comment]
cara existe gay em todas as esferas sociais em todas as profissões creio que toda ou qualquer empresa seja contra preconceito, assunto desnecessArio de ser debatido, beira ao absurdo

**Comment by wagnerbeethoven** [comment]
@fernandoRj, **desnecessário**?, acredito que todo assunto seja válido discussão, ainda mais quando há nítido preconceito com lgbts e estamos falando de brasil, país que mais mata a população gay do mundo. acredito que vc não deva sofrer com isso, mas pratique empatia!

**Comment by camillodev** [comment]
Sou LGBT e nunca trabalhei com outro LGBT na area de TI. E isso é ruim pra mim, porém a gente se adapta. A area de TI é sim muito machista e isso afasta muito outros grupos sociais, e diversidade pode enriquecer muito uma empresa. Parabens pela iniciativa, respondido o questionário.

**Comment by fernandoRj** [comment]
ninguém ta nem ae para a sua orientação sexual eu realmente nao me importo desde que tenha respeito entre todos e inclusive em todos os lugares em que eu trabalhei nunca vi um homosexual ser tratado de maneira diferente por ser homosexual mania essa de querer meter sexualidade em td ate em um lugar que nao tem nada a ver com isso

**Comment by felipefialho** [comment]
@fernandoRj , se o @wagnerbeethoven achou necessário fazer essa pesquisa é porque ela se faz necessária, nada surge do nada. 

Fico contente que você nunca presenciou nenhum desrespeito nos lugares que trabalhou, mas imagino que essa não é a realidade do mercado e o objetivo da pesquisa é justamente levantar questionamentos. 

Não tem porque se incomodar.

**Comment by vsppedro** [comment]
@fernandoRj, podemos aprender algo com a pesquisa. Talvez ela concorde com o seu achismo, que preconceito não existe no mercado, e talvez não concorde mostrando outro cenário. Para isso precisamos deixá-la acontecer. 

Mas vamos concordar que é desnecessário desmerecer a tentativa de coletar depoimentos. Qual mal ela pode fazer?

Sabe como você pode mostrar o seu ponto? Mandando esse link para todos os colaboradores LGBT que você conhece. Dessa forma, com a respostas deles e de outros, iremos saber se de fato eles não são tratados de forma diferente.

@wagnerbeethoven, minha sugestão é deixar esses dados abertos, ocultando os nomes por privacidade, para que as pessoas saibam o quão tóxico pode vir a ser um ambiente de trabalho de TI para um LGBT.

**Comment by wagnerbeethoven** [comment]
@fernandoRj felizmente a sexualidade diz muito de uma pessoa, muda a forma de ver o mundo, já que os não padrões tem tratamento diferenciado. sinceramente não sei qual seu incomodo com a pesquisa, acredito que se não curtiu, como outras três pessoas, põe um dislike, mas isso não reduz a importância ou tão pouco tira a necessidade de tal pesquisa. 

@VSPPedro infelizmente a pesquisa não é minha, dai não tem o que mudar.

@camillodev também nunca trabalhei com LGBT, acredito que isso seja realidade de muita gente no mercado, muitos vivem escondendo a sexualidade.
